### DISCUSSION TITLE
Why is this project seemingly exclusively sponsored by sketchy companies, and proud of that fact?

### DISCUSSION THREAD

**Discussion by era-epoch** [socketio/socket.io#5190] [General]
The library homepage at https://socket.io/ seems to proudly boast about being the tool of choice for online casinos and social media bot-account services ...

Not a good look, IMO. I guess you're happy getting their blood money though ...

**Comment by darrachequesne** [comment]
Hi! That's a fair question, thanks a lot for raising this :+1: 

Actually, I'm not very happy about this, it's gotten really out of hand lately. I am currently looking for other ways to keep the project sustainable and not let it rot without maintenance.
